src/host/getset.cpp

```diff
@@ -1410,6 +1415,10 @@ void DoSrvPrivateAllowCursorBlinking(SCREEN_INFORMATION& screenInfo, const bool
     {
         cursorPosition.X = 0;
     }
+    else
+    {
+        cursorPosition = textBuffer.ClampPositionWithinLine(cursorPosition);

```

Comment:
This is for when a LF moves you into a non-single-width line, right?

Assuming there's no deferred EOL breakage we're concerned with here since this _is_ a cursor movement.

Reply:
Correct. The delayed EOL flag is going to be reset as soon as we set the cursor position, so I don't think this should break anything in that regard.